unfoldingWord/translationCore
# React 16

#### This pull request addresses:
This PR adds React 16 compatibility

#### How to test this pull request:
Basically you need to pull all the submodules besides tC_resources, remove the old node_modules, checkout my branch ```feature-jay-3594``` run npm install and check all subcomponent functionality

I made a script to make the process easier assuming that you have no work thats not already saved on your submodule branches.

**Testing all modules**
```npm run test-all feature-jay-3594```

**Submodule PR's**
https://github.com/translationCoreApps/translationWords/pull/106
https://github.com/translationCoreApps/ScripturePane/pull/83
https://github.com/translationCoreApps/VerseCheck/pull/87
https://github.com/translationCoreApps/translationNotes/pull/82
https://github.com/translationCoreApps/translationHelps/pull/49
https://github.com/translationCoreApps/word-alignment-tool/pull/17


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3650)
